veps/sig-compute/secure-execution.md

```diff
@@ -0,0 +1,60 @@
+# Overview
+Implement support for IBM Z Secure Execution VMs in kubevirt
+
+## Motivation
+To enable customers to deploy secure workloads on their IBM Z machines using kubevirt.
+
+## Goals
+To enable kubevirt to run Secure Execution VMs on s390x
+
+## Non Goals
+Changing how existing VMs run.
+
+## Definition of Users
+- IBM Z users wanting to secure their new VMs using Secure Execution
+- IBM Z users wanting to migrate existing Secure Execution VMs to kubevirt
+
+## User Stories
+(list of user stories this design aims to solve)
+
+## Repos
+https://github.com/kubevirt/kubevirt
+https://github.com/kubevirt/api
+
+# Design
+For implementing this feature, the api would need to be expanded to have a Secure Execution option under Launch Security next to AMD SEV.
+Similiar to SEV it would be first behind a Feature Gate.
+
+Afterwards kubevirt would need to be expanded to handle setting the correct Launch Security type for Secure Execution.
+To achieve this, the virt-launcher pod would currently need to be run as privileged for Secure Execution VMs, as Libvirt checks for `/sys/firmware/uv` when verifying that the Host supports Secure Execution VMs. Unfortunately `/sys/firmware` is intentionally empty in unprivileged pods and kubernetes prevents mounting anything under this path. There are potentially Feature Gates in kubelet to allow this, however i have not yet been able to get this to work. From what i gathered you would need to at least enable `ProcMountType` and `UserNamespacesSupport` to be able to launch a Secure Execution VM inside of an unprivileged pod.
```

Comment:
Is this required or not? I don't really understand from the sentence

Reply:
It is not required, i explain the workaround in the next paragraph. Maybe i should switch it around a bit, first explaining how i am doing it and then why it needs to be done that way?

Reply:
If this is not needed, then you can simply drop it